lib/thrift/transport/http_client_transport.rb

```diff
@@ -60,7 +60,7 @@ def flush
 
       resp = http.post(@url.request_uri, @outbuf, @headers)
       if 'application/x-thrift'.downcase != resp.content_type.downcase
-        raise TransportException.new(TransportException::UNKNOWN, "Unexpected response content type: " + resp.content_type)
+        raise TransportException.new(TransportException::UNKNOWN, "Unexpected response content type: #{resp.content_type} code: #{resp.code}")
```

Comment:
Maybe more consistent with the rest of our apps:
```suggestion
        raise TransportException.new(TransportException::UNKNOWN, "Unexpected response_content_type= #{resp.content_type} code=#{resp.code}")
```